### ISSUE TITLE
Temp file cleanup error due to wrong paths

### ISSUE THREAD

**Issue by doomprophet** [kartik-v/yii2-mpdf#57]
In the Pdf.php file:
```
public function initTempPaths()
    {
        if (empty($this->tempPath)) {
            $this->tempPath = Yii::getAlias('@runtime/mpdf');
        }
        $prefix = $this->tempPath . DIRECTORY_SEPARATOR;
        static::definePath('_MPDF_TEMP_PATH', "{$prefix}tmp");
        static::definePath('_MPDF_TTFONTDATAPATH', "{$prefix}ttfontdata");
    }
```
Right now this creates the fontdata and tmp files in the runtime/mpdf folder and not in runtime/mpdf/tmp or ttfontdata folders. This prevents the cleanup for temp files 1hour old or more and can cause memory problems.
It should be:
```
public function initTempPaths()
    {
        if (empty($this->tempPath)) {
            $this->tempPath = Yii::getAlias('@runtime/mpdf');
        }
        $prefix = $this->tempPath . DIRECTORY_SEPARATOR;
        static::definePath('_MPDF_TEMP_PATH', "{$prefix}tmp".DIRECTORY_SEPARATOR);
        static::definePath('_MPDF_TTFONTDATAPATH', "{$prefix}ttfontdata".DIRECTORY_SEPARATOR);
    }
```
Edit: Changed '/' to DIRECTORY_SEPARATOR

**Comment by kartik-v** [comment]
Check with the updated code from dev-master.

**Comment by doomprophet** [comment]
There doesn't seem to be any changes in the function initTempPaths() in v1.0.2. I suppose the mpdf v7 update solves it. I will give it a try after it becomes a stable release. For the time being I have patched it as above for the v1.0.1

**Comment by kartik-v** [comment]
You are right... reopened isse for enhancement.
